recipes/bip32/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eox pipefail
+
+${PYTHON} -m pip install --use-pep517 . -vvv
```

Comment:
```suggestion
$PYTHON -m pip install . --no-build-isolation \
    --no-deps --ignore-installed --no-index --no-cache-dir -vv --use-pep517
```
If you call pip install from a script instead of from in the recipe (as in the example recipe) you need to add all the args that are added automatically.

Reply:
@carterbox I only understood this yesterday, while working on the PR for `oincurve-feedstock`!